dataprofiler/tests/profilers/test_int_column_profile.py

```diff
@@ -386,7 +386,7 @@ def test_profile_merge(self):
         self.assertAlmostEqual(profiler3.variance,
                                expected_profile.pop('variance'), places=3)
         self.assertEqual(profiler3.mean,expected_profile.pop('mean'))
-        self.assertEqual(profiler3.histogram_selection, 'rice')
+        self.assertEqual(profiler3.histogram_selection, 'doane')
```

Comment:
these provide the same results, bc they were identified in a set, they were randomly being chosen. this is now a fixed order.